apps/andi/lib/andi_web/views/input_types.ex

```diff
@@ -0,0 +1,22 @@
+defmodule AndiWeb.Views.InputTypes do
+  @select "select"
+  @enter "enter"
+
+  @input_types %{
```

Comment:
Reached out on slack. The "enter" comes from the validation tag "Please enter..." under text input fields. This is the default message. "Select" will be used instead of "enter" for any names that appear within this list that has the @select prop.